zutil.c

```diff
@@ -22,16 +22,7 @@ const char * const zng_errmsg[10] = {
     (const char *)""
 };
 
-const char zlibng_string[] =
-    " zlib-ng 1.9.9 forked from zlib 1.2.11 ";
-
-#ifdef ZLIB_COMPAT
-const char * ZEXPORT zlibVersion(void) {
-    return ZLIB_VERSION;
-}
-#endif
-
-const char * ZEXPORT zlibng_version(void) {
```

Comment:
zlibng_version no longer exposed when zlib-compat mode. Is this ok?

Reply:
I think we should not completely hide that we are in fact zlib-ng and not zlib, so we should make it possible for programs to detect this, at least at compile-time.

We need to return ZLIB_VERSION in compat-mode, in case the external application checks the zlib-version and does things depending on that version, but we should use ZLIBNG_VERSION in native mode.

I am also not sure about this change of function name, does zlib call it zVersion instead of zlibVersion?

Reply:
`zVersion` is a preprocessor define. It would only expose `zlibVersion` if in `ZLIB_COMPAT` and `zlibng_Version` otherwise.

Reply:
I have added these functions back.